### PR TITLE
build_targets: add haydn to jenkins

### PR DESCRIPTION
**Pull request by INCENDIOCODM**

### CODE REVIEW

File: build_targets [hunk 1]
```diff
@@ -32,3 +32,4 @@ sweet user yes yes
 vayu user yes yes
 violet userdebug yes yes
 zippo userdebug yes no
+haydn userdebug yes no
```
Comment (gwolf2u) [inline build_targets]: add in proper place first

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: build_targets (hunk 1 above)
Comment (gwolf2u) [reply build_targets]: also
don't clone stuff via vendor.sh call
https://github.com/crdroidandroid/android_device_xiaomi_haydn/blob/13.0/vendorsetup.sh

Comment (gwolf2u) [reply build_targets]: and have a read again here
https://github.com/crdroidandroid/build_targets#dependencies
currently not proper setup

Comment (INCENDIOCODM) [reply build_targets]: I have removed the vendor.sh call
Added the Haydn at proper place 

Could you please tell me 
What's the problem in dependencies

Regards, Vinayak

**Comment by gwolf2u** [comment]
resubmit when ready

**Pull request closed without merging**